.github/workflows/build-test-publish.yml

```diff
@@ -12,12 +12,10 @@ jobs:
             - name: Checkout repository
               uses: actions/checkout@v4
 
-            # Deprecated should be set to the main repository if the update-typo12 branch is merged.
             - name: Checkout DFG-Viewer repository
               uses: actions/checkout@v4
               with:
-                  repository: markusweigelt/dfg-viewer
-                  ref: update-typo12
+                  repository: slub/dfg-viewer
```

Comment:
Fixing this line and also the line below was necessary, but why not using the original code which did not have a hard coded reference to "slub"?

Reply:
See PR #342 which restores the original code.